### DISCUSSION TITLE
Fused Attention

### DISCUSSION THREAD

**Discussion by bearn01d** [intel/intel-extension-for-transformers#724] [Q&A] [answered]
Is Fused Attention as described [here](https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/docs/fused_attention.md) also planned to be implemented for other ISAs, like AVX_VNNI?

**Comment by DDEle** [comment, marked as answer]
Thank you for your interest of the Fused Attention optimization. We are planning on this. However, it will take some time as Fused Attention works only on activation which are more prone to quantization in terms of accuracy. In addition, we need to be more careful in terms of performance as there is no zero-cost weight quantization.

**Reply by bearn01d** [reply]
Sounds good, thanks for your reply!
